controlplane/kubeadm/controllers/controller_test.go

```diff
@@ -175,7 +175,7 @@ func TestReconcileUpdateObservedGeneration(t *testing.T) {
 		errGettingObject = testEnv.Get(ctx, util.ObjectKey(kcp), kcp)
 		g.Expect(errGettingObject).NotTo(HaveOccurred())
 		return kcp.Status.ObservedGeneration
-	}, 10*time.Second).Should(Equal(generation))
+	}, 60*time.Second).Should(Equal(generation))
```

Comment:
this was causing failures in my tests.

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/4466